en_US/data-integration/data-bridge-mqtt.md

```diff
@@ -73,24 +73,29 @@ The following section will use EMQX [public MQTT broker](https://www.emqx.com/en
      - **Remote MQTT Broker**: Subscribe to the remote topics.
      
        - **Topic**: In cluster mode, we can use the [Shared Subscription](../mqtt/mqtt-shared-subscription.md) to avoid repeated messages, therefore we will fill in `$share/g/remote/topic/ingress`
-       - QoS: Select `0`.
+       - **QoS**: Select `0`.
      
      - **Local MQTT Broker**: Forward the received messages to specific local topics or leave them blank, then these messages will first be processed by the configured rules and then forwarded with the [republish action](./rules.md).
        - **Topic**: Input `local/topic/ingress`.
        - **QoS**: Select `0` or `${qos}` (to use the QoS of the received messages).
        - **Retain**: Confirm whether the message will be published as a retained message.
        - **Payload**: Payload template for the messages to be forwarded, and Supports reading data using `${field}` syntax.
+
+     - **Pool size**: Specify the size of the pool of MQTT clients that will ingest messages from the remote broker, we'll set `8` to make use of available parallelism. This is safe as long as shared subscription is used for the remote topic.
```

Comment:
```suggestion
     - **Pool size**: Introduced since 5.1, this config is to specify the size of the pool of MQTT clients that will ingest messages from the remote broker, we'll set `8` to make use of available parallelism. This is safe as long as shared subscription is used for the remote topic.
```